datadog_checks_dev/setup.py

```diff
@@ -36,6 +36,7 @@
     "shutilwhich==1.1.0; python_version < '3.0'",
     "subprocess32==3.5.4; python_version < '3.0'",
     'tenacity',
+    'jsonschema>=3.2.0',
```

Comment:
Can we remove it from here then? https://github.com/DataDog/integrations-core/pull/7322/files#diff-99757cb9f5bc9cf74cf195fa4128c105R79